src/EventBuilder/EventBuilder.csproj

```diff
@@ -1,26 +1,30 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
-<PropertyGroup>
-  <OutputType>Exe</OutputType>
-  <TargetFramework>net461</TargetFramework>
-  <AssemblyName>EventBuilder</AssemblyName>
-  <RootNamespace>EventBuilder</RootNamespace>
-</PropertyGroup>
-
-
-<ItemGroup>
-  <Content Include="DefaultTemplate.mustache">
-    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-  </Content>
-</ItemGroup>
-
-<ItemGroup>
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net461</TargetFramework>
+    <AssemblyName>EventBuilder</AssemblyName>
+    <RootNamespace>EventBuilder</RootNamespace>
+  </PropertyGroup>
+  
+  
+  <ItemGroup>
+    <Content Include="DefaultTemplate.mustache">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+  
+  <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="1.9.71" />
     <PackageReference Include="Microsoft.Web.Xdt" Version="2.1.1" />
     <PackageReference Include="Mono.Cecil" Version="0.9.6.1" />
-    <PackageReference Include="NuGet.Core" Version="2.10.1" />
     <PackageReference Include="Nustache" Version="1.15.3.7" />
     <PackageReference Include="Polly" Version="3.0.0" />
     <PackageReference Include="Serilog" Version="1.5.14" />
+    <PackageReference Include="NuGet.Core" Version="2.14.0" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Update="XamarinEssentialsTemplate.mustache">
```

Comment:
This one is None, other mustache above is Content, can we keep them the same type and put them into the same ItemGroup?

Reply:
can do!